### DISCUSSION TITLE
Player API .GetRecentlyPlayed() does not .NextPage() properly

### DISCUSSION THREAD

**Discussion by benlane2003** [JohnnyCrazy/SpotifyAPI-NET#708] [Q&A]
When using the .NextPage() call on the List of PlayHistoryItems that is returned from the Player.GetRecentlyPlayed() call, the items returned are always empty, limiting me to only 50 PlayHistoryItems, I want to be able to .NextPage() them and get more. I'm assuming this is a bug because everything else does .NextPage() properly.

**Comment by JohnnyCrazy** [comment]
Hi,

How are you sure that you even have more than 50 recently played? Did you try to reconstruct the problem in the web api console? Pretty sure its not a bug on this project

**Comment by JohnnyCrazy** [comment]
It seems like the "total" field is always empty...

**Comment by benlane2003** [comment]
The problem is that I cannot find a way to do a NextPage() on the web console, do you know if there is a way to test a page flip on the web console? But yes I am very very sure I have more than 50 items, I've also tested it on multiple accounts, all the same.

**Comment by JohnnyCrazy** [comment]
What I meant is that spotify is limiting the maximum amount of tracks you can get with `GetRecenlyPlayed`, which is maybe 50.

**Reply by benlane2003** [reply]
But the item returned is a Paging item, right? So I assumed, just like all other paging items, it can be paged but it seems like this specific one can't page at all - or rather, when paged it always returns an empty page - it is strange because in the Spotify App itself, when I look at my Recently Played page, there are DEFINITELY more than 50 items. I might post this issue onto the Spotify For Developers Community as a topic.

**Reply by benlane2003** [reply]
Update, there have been topics about this issue posted on the Community since 2020, not a single reply from Spotify.

**Reply by JohnnyCrazy** [reply]
Yeah don't expect one. They're writing stuff like "give us feedback on the developer forums" for ages but actually ignore every bug report there.
